modules/core/src/v2/coins/dot.ts

```diff
@@ -238,7 +252,10 @@ export class Dot extends BaseCoin {
    */
   async signTransaction(params: SignTransactionOptions): Promise<SignedTransaction> {
     const { txHex, prv } = this.verifySignTransactionParams(params);
-    const factory = accountLib.register(this.getChain(), accountLib.Dot.TransactionBuilderFactory);
+    
+    const material = await this.materialDataLookup();
```

Comment:
The only thing Im not 100% sure on here is the fact that  we're calling an endpoint when signing a transaction. I believe OVC will error out here when it tries to sign.

Do I have that right @Logicwax ?

Reply:
OVC is offline.  And so cannot do this.

Reply:
OVC is offline.  And so cannot do this.